dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/HttpUtils.java

```diff
@@ -138,7 +138,7 @@ public static List<Locale> parseAcceptLanguage(String header) {
     public static List<Locale> parseContentLanguage(String header) {
         List<Locale> locales = new ArrayList<>();
         if (header == null) {
```

Comment:
Can I move this line to line 143? This will avoid creating unnecessary objects

Reply:
OK